app/App.css

```diff
@@ -0,0 +1,6 @@
+.main {
```

Comment:
we shouldn't need files like this with tailwind, you can instead add these in the classname of the div
```
<div class="ml-[10%] mr-[10%] mt-[10%] mb-[10%]">
  <!-- Content goes here -->
</div>
```